README.md

```diff
@@ -1,16 +1,27 @@
-# datadog-mobile-app-run-tests
+# Datadog Continuous Testing for Bitrise
```

Comment:
Nice! It's the full name indeed ([docs](https://docs.datadoghq.com/continuous_testing/)), and we use that kind of title for Azure DevOps: https://github.com/DataDog/datadog-ci-azure-devops

Should we align that title for all CI integrations? cc. @jakepruitt

Reply:
Yup i read the other 3 integration READMEs and Frankenstein-ed this one from the best parts of them in my opinion :D